core/src/saros/net/xmpp/XMPPConnectionService.java

```diff
@@ -272,6 +284,15 @@ public synchronized void connect(
     }
   }
 
+  private void sendAvailablePresenceWithClientIdentifier() {
+    String ver = EntityCapsManager.getInstanceFor(connection).getCapsVersion();
+    CapsExtension caps = new CapsExtension(XMPP_CLIENT_IDENTIFIER, ver, "sha-1");
```

Comment:
Move hashing algorithm choice to static constant to make it easier to notice when looking at the class?

Reply:
done